keyboards/ergodash/rev2/keymaps/greenshadowmaker/keymap.c

```diff
@@ -0,0 +1,156 @@
+    KC_LSFT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,           XXXXXXX,     XXXXXXX,          XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_RSFT, \
+    XXXXXXX, XXXXXXX, XXXXXXX,XXXXXXX,             LOWER,   KC_SPC ,XXXXXXX,     XXXXXXX, KC_SPC ,   RAISE,          XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX   \
+  ),
+
+  /* Adjust  (Both Raise and Lower Together)
+   * ,----------------------------------------------------------------------------------------------------------------------.
+   * | Reset|      |      |      |      |      |      |                    |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------+------+--------------------+------+------+------+------+------+------+------|
+   * |      |      |      |      |rgbtog|bl_tog|      |                    |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------+------+--------------------+------+------+------+------+------+------+------|
+   * | Raise|      |rgbhui|rgbsai|rgbvai|bl_inc|      |                    |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------+---------------------------+------+------+------+------+------+------+------|
+   *|EEP_RST|      |rgbhud|rgbdec|rgbvad|bl_dec||||||||      |      |      ||||||||      |      |      |      |      | Shift|
+   * |-------------+------+------+------+------+------+------+------+------+------+------+------+------+------+-------------|
+   * |      |      |      |      |||||||| Lower| Space|      ||||||||      | Space| Raise||||||||      |      |      |      |
+   * ,----------------------------------------------------------------------------------------------------------------------.
+   */
+  [_ADJUST] = LAYOUT(
+      RESET, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,  \
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, RGB_TOG, BL_TOGG, XXXXXXX,                        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, \
+      RAISE, XXXXXXX, RGB_HUI, RGB_SAI, RGB_VAI,  BL_INC, XXXXXXX,                        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, \
+    EEP_RST, XXXXXXX, RGB_HUD, RGB_SAD, RGB_VAD,  BL_DEC,           XXXXXXX,     XXXXXXX,          XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_RSFT, \
+    XXXXXXX, XXXXXXX, XXXXXXX,XXXXXXX,             LOWER,   KC_SPC ,XXXXXXX,     XXXXXXX,  KC_SPC,   RAISE,          XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX   \
+  )
+};
+
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
```

Comment:
If you are only have the one default layer, you should probably remove this.

Reply:
Just the QWERTY one?  I tried taking out the entire 

bool process_record_user(

But Layer switch didn't work after

Reply:
I haven't figured out where the tri_layer magic all comes from

Reply:
it comes from quantum.h, actually. 

And yeah, without the RAISE and LOWER keycodes, the layers won't work. 

https://docs.qmk.fm/#/ref_functions?id=olkb-tri-layers